src/navigation/underline-nav.scss

```
@@ -22,8 +22,7 @@
   background-color: transparent;
   border: 0;
   // stylelint-disable-next-line primer/borders
-  border-bottom: 2px $border-style transparent;
-  transition: border-bottom-color 0.36s ease-in;
+  border-bottom: 2px $border-style rgba($border-gray-dark, 0);
```

Fix this after the merge

```suggestion
  border-bottom: 2px $border-style transparent;
```